docs/af5-app-from-scratch/modules/ROOT/pages/implement-command-subscribe-student.adoc

```diff
@@ -0,0 +1,603 @@
+:navtitle: Feature: Subscribe Student
+:reftext: Implementing the subscribe student feature
+
```

Comment:
The header here is missing, this causes the menu to not show the title when antora is built